hack/verify_boilerplate.py

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env python3
 
-# Copyright 2015 The Kubernetes Authors.
```

Comment:
🤨

Reply:
agree with @BenTheElder we should not drop the year here :)

Reply:
Hehe, this broke the CI, let me check if I can re-enable it.